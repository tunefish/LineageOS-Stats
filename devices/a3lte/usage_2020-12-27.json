{
  "14.1-20171111-nightly-a3lte": 3,
  "15.1-20180518-unofficial-a33g": 113,
  "15.1-20180518-unofficial-a3lte": 138,
  "15.1-20180518-unofficial-a3ulte": 257,
  "15.1-20201128-unofficial-herolte": 1,
  "16.0-20181122-unofficial-a3lte": 164,
  "17.1-20200715-unofficial-tissot": 8,
  "17.1-20200716-unofficial-herolte": 5,
  "17.1-20200720-unofficial-a3lte": 51,
  "17.1-20200819-unofficial-a3lte": 404,
  "17.1-20200821-unofficial-z3": 19,
  "17.1-20201112-unofficial-ef63": 1,
  "17.1-20201116-unofficial-z3": 1,
  "17.1-20201122-unofficial-herolte": 1,
  "17.1-20201222-unofficial-a3lte": 4,
  "18.1-20200819-official-a3lte": 1
}