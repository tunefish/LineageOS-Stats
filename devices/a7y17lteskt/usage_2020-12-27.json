{
  "17.1-20200715-unofficial-tissot": 1466,
  "17.1-20200821-unofficial-z3": 970
}