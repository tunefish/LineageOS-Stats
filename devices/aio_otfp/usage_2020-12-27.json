{
  "13.0-20170114-unofficial-aio_otfp": 1,
  "14.1-20170118-unofficial-aio_otfp": 1,
  "14.1-20170214-unofficial-aio_otfp": 63,
  "14.1-20170312-unofficial-aio_otfp": 4,
  "14.1-20170408-unofficial-aio_otfp": 15,
  "14.1-20170420-unofficial-aio_otfp": 3,
  "14.1-20170430-unofficial-aio_otfp": 4,
  "14.1-20170616-unofficial-aio_otfp": 2,
  "14.1-20170617-unofficial-aio_otfp": 3,
  "14.1-20170702-unofficial-aio_otfp": 6,
  "14.1-20170729-unofficial-aio_otfp": 1,
  "14.1-20170820-unofficial-aio_otfp": 3,
  "14.1-20171031_105651-unofficial-aio_otfp": 1,
  "14.1-20171203_053532-unofficial-aio_otfp": 13,
  "14.1-20171216_141341-unofficial-aio_otfp": 18,
  "14.1-20180127-official-aio_otfp": 30,
  "14.1-20180127-unofficial-aio_otfp": 182,
  "14.1-20200109-unofficial-aio_otfp": 6,
  "15.1": 5,
  "15.1-20180221-official-aio_otfp": 1,
  "15.1-20180221-unofficial-aio_otfp": 429,
  "15.1-20180309-unofficial-aio_otfp": 17,
  "15.1-20180421-unofficial-aio_otfp": 36,
  "15.1-20180509-unofficial-a7000": 9,
  "15.1-20180509-unofficial-aio_otfp": 33,
  "15.1-20180509-unofficial-aio_row": 11,
  "15.1-20180509-unofficial-p6000": 4,
  "15.1-20180511-unofficial-aio_otfp": 9,
  "15.1-20180627-unofficial-aio_otfp": 2,
  "15.1-20180713-unofficial-aio_otfp": 69,
  "7.0-20190124-unofficial-aio_otfp": 8,
  "bobs-1.1.--190924-amateur": 2,
  "lineage_gemini-userdebug 7.1.2 njh47f 8290c49eea test-keys": 15,
  "lineageos-15.1-by-tanish2k09": 1
}