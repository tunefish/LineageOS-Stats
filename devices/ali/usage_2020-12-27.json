{
  "16.0-20190621-unofficial-ali": 97,
  "16.0-20190622-unofficial-ali": 4,
  "16.0-20190623-unofficial-ali": 11,
  "16.0-20190625-unofficial-ali": 172,
  "17.1-20200513-unofficial-ali": 8,
  "17.1-20200518-unofficial-ali": 34,
  "17.1-20200726-unofficial-ali": 35,
  "17.1-20200802-unofficial-ali": 69,
  "17.1-20201001-unofficial-ali": 563,
  "17.1-20201225-unofficial-ali": 9,
  "18.1-20201223-unofficial-ali": 4,
  "18.1-20201224-unofficial-ali": 8
}