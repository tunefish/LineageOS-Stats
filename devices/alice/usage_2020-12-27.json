{
  "13.0-20170417-openkirin-alice": 16,
  "13.0-20170418-openkirin-alice": 9,
  "13.0-20170501-openkirin-alice": 2,
  "13.0-20170802-unofficial-alice": 58,
  "14.1-20170417-openkirin-alice": 5,
  "14.1-20170508-unofficial-alice": 2,
  "14.1-20171015-openkirin-alice": 9,
  "14.1-20171130_210808-unofficial-alice": 2,
  "14.1-20171231_141649-official-alice": 1,
  "14.1-20171231_141649-unofficial-alice": 146,
  "14.1-20180525-unofficial-alice": 1,
  "14.1-20180802-unofficial-alice": 1,
  "14.1-20180911-unofficial-alice": 3,
  "14.1-20181001-unofficial-alice": 21,
  "14.1-20181003-unofficial-alice": 2,
  "14.1-20181209-unofficial-alice": 3,
  "14.1-20181215-darkjoker360-alice": 20,
  "14.1-20190203-unofficial-alice": 2,
  "14.1-20190302-darkjoker360-alice": 8,
  "14.1-20190328-darkjoker360-alice": 5,
  "14.1-20190412-darkjoker360-alice": 3,
  "14.1-20190426-darkjoker360-alice": 5,
  "14.1-20190519-darkjoker360-alice": 8,
  "14.1-20190616-darkjoker360-alice": 48,
  "14.1-20190806-unofficial-alice": 8,
  "14.1-20190906-unofficial-alice": 17,
  "14.1-20191119-unofficial-alice": 6,
  "14.1-20191224-unofficial-alice": 11,
  "14.1-20200215-unofficial-alice": 14,
  "14.1-20200329-unofficial-alice": 15,
  "14.1-20200415-unofficial-alice": 5,
  "14.1-20200505-unofficial-alice": 29,
  "14.1-20200616-unofficial-alice": 26,
  "14.1-20200726-unofficial-alice": 14,
  "14.1-20200817-unofficial-alice": 1,
  "14.1-20200818-official-alice": 1,
  "14.1-20200818-unofficial-alice": 327,
  "14.1-20200910-unofficial-alice": 1,
  "14.1-20201017-unofficial-alice": 1,
  "14.1-20201022-unofficial-alice": 1,
  "14.1-20201029-unofficial-alice": 114,
  "14.1-20201221-unofficial-alice": 1,
  "15.1-20191201-darkjoker360-alice": 3,
  "15.1-20200217-darkjoker360-alice": 80,
  "15.1-20200303-darkjoker360-alice": 1,
  "15.1-20200902-unofficial-alice": 1,
  "lineage_alice-userdebug 7.1.2 njh47f 0f2f1c5b17 test-keys": 1
}