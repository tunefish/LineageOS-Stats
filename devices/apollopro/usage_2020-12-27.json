{
  "17.1-20201122-unofficial-apollopro": 1,
  "17.1-20201124-unofficial-apollopro": 5,
  "17.1-20201126-unofficial-apollopro": 7,
  "17.1-20201127-unofficial-apollopro": 5,
  "17.1-20201128-unofficial-apollopro": 1053,
  "17.1-20201201-unofficial-apollopro": 1,
  "17.1-20201203-unofficial-apollopro": 1,
  "17.1-20201212-unofficial-apollopro": 392,
  "17.1-20201220-unofficial-apollopro": 8,
  "17.1-20201221-unofficial-apollopro": 2,
  "17.1-20201223-unofficial-apollopro": 6,
  "17.1-20201224-unofficial-apollopro": 2,
  "17.1-20201226-unofficial-apollopro": 11,
  "18.0-20201222-unofficial-apollopro": 1,
  "18.0-20201223-unofficial-apollopro": 1,
  "treble_arm64_bgn-userdebug 10 qq3a.200905.001 eng.diust.20201122.154631 test-keys": 8
}