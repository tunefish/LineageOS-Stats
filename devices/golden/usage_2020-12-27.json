{
  "13.0-20170118-unofficial-golden": 1,
  "13.0-20170121-unofficial-golden": 1,
  "13.0-20170129-unofficial-golden": 92,
  "13.0-20170224-unofficial-golden": 5,
  "13.0-20170302-unofficial-golden": 8,
  "13.0-20170310-unofficial-golden": 23,
  "13.0-20170328-unofficial-golden": 784,
  "13.0-20170420-unofficial-golden": 53,
  "14.1-20170122-unofficial-golden": 265,
  "14.1-20170318-unofficial-golden": 5,
  "14.1-20170402-unofficial-golden": 10,
  "14.1-20170415-unofficial-golden": 4,
  "14.1-20170419-unofficial-golden": 5,
  "14.1-20170503-unofficial-golden": 1,
  "14.1-20170507-unofficial-golden": 17,
  "14.1-20170522-unofficial-golden": 1,
  "14.1-20170609-unofficial-golden": 479,
  "14.1-20170610-unofficial-golden": 8,
  "14.1-20170628-unofficial-golden": 770,
  "14.1-20170724-unofficial-golden": 46,
  "14.1-20170817-unofficial-golden": 32,
  "14.1-20170920-unofficial-golden": 8,
  "14.1-20171011-unofficial-golden": 72,
  "14.1-20171215_192311-official-golden": 248,
  "14.1-20171215_192311-unofficial-golden": 886,
  "14.1-20180109_002536-unofficial-golden": 195,
  "14.1-20181014-unofficial-golden": 3,
  "14.1-20200817_071197-official-golden-rogerwed": 1,
  "15.1-20181030-unofficial-golden": 5,
  "17.1-20201113-unofficial-tissot": 2,
  "golden-user 8.1.0 oe632e 3de271a2ce release-keys": 1,
  "njh47f release-keys": 1
}